.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2.0
+
+references:
+
+  build_containers_config: &build_container_config
+    docker:
+    - image: circleci/ruby:2.5.1
+      environment:
+        GITHUB_TEAM_NAME_SLUG: apply-for-legalaid
```

Comment:
Just noticed this env variable, is it actually required for this image?

Reply:
i think it was used at image level because multiple steps can perhaps share it rather than keep defining it at step level for each step

Reply:
Not marking it as a blocker, but if that's the intent, that should still be defined in a shared scope that contains the environment variable(s) that are shared across the different jobs/steps. The ruby image itself does not require it so I don't think it should be there.